sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -544,13 +463,11 @@ export class MessageSender extends LinkEntity {
       }
       if (shouldReopen) {
         await defaultLock.acquire(this.senderLock, () => {
-          const options: SenderOptions = this._createSenderOptions({
-            newName: true
-          });
+          const senderOptions = this._createSenderOptions(Constants.defaultOperationTimeoutInMs);
```

Comment:
It looks like we used to set `newName` to true here. Is there a reason we aren't now?

Reply:
Hmmm.. good catch, added it back.